recipes/sentencepiece/build.sh

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_INSTALL_LIBDIR=$PREFIX/lib \
+    -S ..
+make -j $(nproc)
+make install
+
+if [[ "$target_platform" == linux* ]]; then
+  ldconfig -v
+elif [[ "$target_platform" == "osx-64" ]]; then
+  update_dyld_shared_cache
+fi
+
```

Comment:
add `export LD_LIBRARY_PATH=${PREFIX}/lib:$LD_LIBRARY_PATH` here since the Python build requires the sentencepiece libraries.

Reply:
Also add:
```
export CPATH=${PREFIX}/include
export INCLUDE=${PREFIX}/include
export LIBRARY_PATH=${PREFIX}/lib
```